addons/test_mail/tests/test_mail_composer.py

```diff
@@ -32,6 +33,17 @@ def setUpClass(cls):
             name='Eglantine Employee', signature='--\nEglantine')
         cls.partner_employee_2 = cls.user_employee_2.partner_id
 
+        # User without the group "mail.group_mail_template_editor"
+        cls.user_rendering_restricted = mail_new_test_user(
+            cls.env, login='user_rendering_restricted',
+            groups='base.group_user',
+            company_id=cls.company_admin.id,
+            name='Code Template Restricted User',
+            notification_type='inbox',
+            signature='--\nErnest'
```

Comment:
Strange signature for someone names "Code Template Restricted User". Note that I think his or her childhood was not that easy.

Reply:
> Strange signature for someone names "Code Template Restricted User". Note that I think his or her childhood was not that easy.

That would be why they try to use their middle name in normal discussions (and thus as signature).